task_8_9_8.py

```diff
@@ -0,0 +1,49 @@
+from utils import print_intro
+
+print_intro()
+
+
+def validate_number_input(prompt):
+    while True:
+        try:
+            return float(input(prompt))
+        except ValueError:
+            print('Invalid input. Please enter a valid number.')
```

Comment:
so, you could replace **validate_number_input** from utils by this function.
just make cast function as parameter, so in such case you could use the same function for enter int and float.